logging/google/cloud/logging/handlers/handlers.py

```diff
@@ -20,7 +20,11 @@
 
 DEFAULT_LOGGER_NAME = 'python'
 
-EXCLUDED_LOGGER_DEFAULTS = ('google.cloud', 'oauth2client')
+EXCLUDED_LOGGER_DEFAULTS = (
```

Comment:
Nope, if they are, they aren't using it as part of this library and we shouldn't exclude it.

Actually, I'm really curious about this whitelist - I feel like it should be pretty narrow (`google.cloud.logging.handlers`) instead of being so broad. I'm wondering why @waprin picked `google.cloud`.